packages/appeals-service-api/src/lib/notify.js

```diff
@@ -0,0 +1,69 @@
+const { NotifyClient } = require('notifications-node-client');
+const fetch = require('node-fetch');
+const config = require('./config');
+const logger = require('./logger');
+const { getLpa } = require('../services/lpa.service');
+
+function getAddress(siteAddress) {
+  let address = '';
+  if (siteAddress.addressLine1) {
+    address = `${siteAddress.addressLine1}\n`;
+  }
+  if (siteAddress.addressLine2) {
+    address = `${address}${siteAddress.addressLine2}\n`;
+  }
+  if (siteAddress.town) {
+    address = `${address}${siteAddress.town}\n`;
+  }
+  if (siteAddress.county) {
+    address = `${address}${siteAddress.county}\n`;
+  }
+  if (siteAddress.postcode) {
+    address = `${address}${siteAddress.postcode}`;
+  }
+  return address;
+}
+
+async function sendEmail(appeal) {
+  logger.info(`Sending email...`);
+  logger.debug(`appeal = \n${JSON.stringify(appeal, null, 2)}`);
```

Comment:
This needs redacting from prod for GDPR

Reply:
@a6software Good call. Probably best just to put `logger.debug({ appealId: appeal.id }, 'Some message')`

Reply:
I am not sure what you mean here. Usually managing access to and utilisation of log data together policy decisions about their retention is sufficient for GDPR compliance.